src/Lime.Messaging/Contents/Reply.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Runtime.Serialization;
+using Lime.Protocol;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Lime.Messaging.Contents
+{
+    /// <summary>
+    /// Represents a reply message.
+    /// </summary>
+    [DataContract]
+    public class Reply : Document
+    {
+        public static readonly string MimeType = "application/vnd.lime.reply+json";
+        public static readonly MediaType MediaType = MediaType.Parse(MimeType);
+
+        public const string CONTENT = "content";
+        public const string REPLY_TO = "replyTo";
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Reply"/> class.
+        /// </summary>
+        public Reply()
+            : base(MediaType)
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets the content of the reply.
+        /// </summary>
+        [DataMember(Name = CONTENT)]
+        public DocumentContainer Content { get; set; }
```

Comment:
Suggestion: I think it would be nice to change the content of the reply, to `ReplyContent`, so that it doesn't look like this:
`"content":{ "content":{...}, ... }`